docs/root/configuration/http_filters/router_filter.rst

```diff
@@ -99,6 +99,9 @@ retriable-status-codes
   in either :ref:`the retry policy <envoy_api_field_route.RouteAction.RetryPolicy.retriable_status_codes>`
   or in the :ref:`config_http_filters_router_x-envoy-retriable-status-codes` header.
 
+rate-limited
+  Envoy will attempt a retry if a request is rate limited (with response code 429) by the :ref:`rate limit service <arch_overview_rate_limit>`.
```

Comment:
I think I would remove the "by the rate limit service" clause? Since really you are just adding a 429 retry policy?

Do we want to add a parallel grpc response code policy for resource exhausted or something like that?

Reply:
> I think I would remove the "by the rate limit service" clause? Since really you are just adding a 429 retry policy?
Yes. I will remove that.
> Do we want to add a parallel grpc response code policy for resource exhausted or something like that?
I think parallel grpc response code policy is not required because rate limiting filter returns 429 status (along with gRPC status `UNAVAILABLE`). So  the newly added policy should be sufficient. Am I missing some thing here?

Reply:
I also realized the condition is not doing what I am intending to do (and the corresponding test also). Now I have corrected. It should be clear now. PTAL.